src/components/radio/RadioSelect.tsx

```diff
@@ -1,19 +1,49 @@
 import * as React from 'react';
+import * as _ from 'underscore';
 import {ToggleForm} from '../childForm/ToggleForm';
 import {Radio} from './Radio';
 
 export interface IRadioSelectProps {
+    id?: string;
     name?: string;
-    value?: string;
     disabled?: boolean;
-    onChange?: (value: string) => void;
     children?: Array<React.ReactElement<Radio>> | Array<React.ReactElement<ToggleForm>>;
+    /**
+     * used in RadioSelectConnected component only

```

Comment:
The comment could probably be removed, why not let the user decide if they need this attribute? :)

If you don't want to remove it, is this strictly the only use case where this attribute is necessary? If so, I would split the prop into a different interface so that standard props don't include it, like `IRadioSelectConnectedProps`.